packages/frontend-2/plugins/001-logger.ts

```diff
@@ -293,6 +314,15 @@ export default defineNuxtPlugin(async (nuxtApp) => {
     })
   }
 
+  // Log important tier app mounted msg for seq
+  if (!import.meta.server) {
+    nuxtApp.hook('app:mounted', () => {
+      logger.info('App mounted in the client', {
```

Comment:
example - now in Seq we can correlate if SSR renders also lead to full hydration and load in the browser, as opposed to SSR render being triggered by curl or something and no clientside JS actually being loaded